src/Stryker.CLI/Stryker.CLI/CLIOptions.cs

```diff
@@ -46,6 +46,14 @@ public static class CLIOptions
             JsonKey = "timeout-ms"
         };
 
+        public static readonly CLIOption<string[]> ExcludedMutations = new CLIOption<string[]>
+        {
+            ArgumentName = "--excluded-mutations",
+            ArgumentShortName = "-em <mutator-name>",
+            ArgumentDescription = "The given mutators will be excluded for this mutation testrun.",
```

Comment:
Imo the description for this feature should give some hint as to where you can find the list of available mutators. Preferably we should add a commandline argument that doesn't run stryker but simply lists the mutators available in this version of stryker and a link to github. For now I think it would be enough to add the link to the readme in the description.

Reply:
We have the MutatorType enum, we could just print it here. I don't know if an extra command would be what we want.

Reply:
I suppose that could work as well. I was thinking the list-mutators command
would include a detailed description and then the help text for excluded
mutations could mention using the list-mutators command to find out which
mutations to exclude.

Reply:
I've thought about this and for now I don't think this is needed for now. We can always add this as a new feature request later.